pkg/migrations/op_add_column_test.go

```diff
@@ -152,7 +152,7 @@ func TestAddColumn(t *testing.T) {
 			wantStartErr: migrations.BackfillNotPossibleError{Table: "users"},
 		},
 		{
-			name: "add serial columns",
+			name: "add serial columns with generated column",
```

Comment:
Should we change this test or add a new one just for generated columns?

Reply:
I added a separate test as you requested.